src/main/java/uk/gov/hmcts/reform/feature/config/SecurityConfiguration.java

```diff
@@ -66,10 +76,24 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .anyRequest().hasRole(ROLE_ADMIN)
                 .and()
-                .httpBasic()
+                .exceptionHandling().authenticationEntryPoint(new EntryPoint())
                 .and()
                 .csrf().disable();
         }
+
+        private static class EntryPoint implements AuthenticationEntryPoint {
```

Comment:
As discussed we can move it to it's own class instead of inner class.Can be done as this PR or separate one.